scripts/variantstore/wdl/GvsAoUReblockGvcf.wdl

```diff
@@ -77,7 +81,7 @@ task ReblockAndCopy {
     set -x
 
     if [ ~{site_id} -a ~{dir} == "null" ]; then
-      echo "dir is not set to a valid value: ~{dir}. check site_id - only valid values are ['bi', 'bcm', 'uw']"
+      echo "dir is not set to a valid value. site_id is ~{site_id}. check site_id - only valid values are ['bi', 'bcm', 'uw']"
```

Comment:
nit for clarity (feel free to finesse)
```suggestion
      echo "The additional output directory for the reblocked gVCFs can not set to a valid value because the site_id is '~{site_id}' and the only valid values are ['bi', 'bcm', 'uw']."
```

Reply:
thank you! this is way better